data_sources/Youth Pass Municipality Contacts and Zip Codes.csv

```diff
@@ -84,6 +84,8 @@
 02238,Cambridge,tpass@cambridgema.gov,617-349-6234,32169,FALSE,"City of Cambridge, Department of Human Service Programs",51 Inman St,"Cambridge, MA 02139","Hours: Monday, 8:30 AM – 8 PM; Tuesday – Friday, 8:30 AM – 5 PM","Please email or call to schedule a time to pick up your Youth Pass. Once your appointment is confirmed, visit the Cambridge Youth Pass office to pick up your card:"
 02150,Chelsea,youthpass@greenrootschelsea.org,617-466-3076 ext. 123,32156,TRUE,GreenRoots,"90 Everett Ave, 3rd Floor, Suite 10","Chelsea, MA 02150","Hours: Monday – Thursday, 9 AM – 5 PM","Please email or call to schedule a time to pick up your Youth Pass. Once your appointment is confirmed, visit the Chelsea Youth Pass office to pick up your card:"
 02149,Everett,andrea.romboli@ci.everett.ma.us,617-394-2323,37023,FALSE,"City of Everett, Department of Health and Human Services","Connolly Center, 90 Chelsea St","Everett, MA 02149","Hours: Monday, 8 AM – 4 PM; Tuesday – Thursday, 8 AM – 5 PM; Friday, 8 AM – 12 PM","Please email or call to schedule a time to pick up your Youth Pass. Once your appointment is confirmed, visit the Everett Youth Pass office to pick up your card:"

```

Comment:
Suggestion: We can remove lines 87-88 here. We just need to change line 86 to list Catherine Connor's contact email instead of Andrea Romboli's (assuming I'm reading the asana task correctly)

```suggestion
02149,Everett,catherine.connor@ci.everett.ma.us,617-394-2323,37023,FALSE,"City of Everett, Department of Health and Human Services","Connolly Center, 90 Chelsea St","Everett, MA 02149","Hours: Monday, 8 AM – 4 PM; Tuesday – Thursday, 8 AM – 5 PM; Friday, 8 AM – 12 PM","Please email or call to schedule a time to pick up your Youth Pass. Once your appointment is confirmed, visit the Everett Youth Pass office to pick up your card:"

```

Reply:
ditto: https://github.com/mbta/reduced_fares/pull/357/commits/ed53461e5d486b3b4142c6b1c97427eb9eb14ca2